packages/ui/VirtualList/VirtualListBasic.js

```diff
@@ -993,7 +1000,7 @@ class VirtualListBasic extends Component {
 
 	// For individually sized item
 	adjustItemPositionWithItemSize () {
-		if (this.itemContainerRef.current) {
+		if (this.contentRef.current) {
```

Comment:
I think this one is not identical to the old condition.
Maybe `cc.length` would be better.